frontends/web/src/routes/account/send/feetargets.tsx

```diff
@@ -38,10 +38,12 @@ export type Props = FeeTargetsProps & TranslateProps;
 
 interface FeeTarget {
     code: Code;
+    feeRateInfo: string;
 }
 interface State {
     feeTargets: FeeTarget[] | null;
     feeTarget?: string | null;
+    expert?: boolean;
```

Comment:
Here's where I think `config.frontend.expertFee` would've made it slightly easier. Backend doesn't really care what the UI shows. If the backend receives `custom` fee target, it computes custom. Otherwise, one of the standard. And the UI shows what it needs to show.